applications/FluidDynamicsApplication/symbolic_generation/two_fluid_navier_stokes/two_fluid_navier_stokes_element.lyx

```diff
@@ -0,0 +1,1274 @@
+#LyX 2.2 created this file. For more info see http://www.lyx.org/
```

Comment:
i will not insist on this, however if you essentially copy this to a ipynb file, than you can see it in the github

Reply:
Is there any easy and fast way to do this?